src/sbt-test/sbt-plugins/simple/test

```diff
@@ -0,0 +1,29 @@
+# See http://eed3si9n.com/testing-sbt-plugins for sbt-scripted test syntax
+# initialize a git repository to make our plugins happy
+$ exec git init
+$ exec git add .
+$ exec git commit -m "initial commit"
+
+> checkFormat
+
+# make sure all our code compiles, including:
+# core/src/it test code that depends on core/src/test code
+> it:compile
+
+# Style plugin tests
+> checkStyle
+> styleCheckStrict
+-> test:styleCheckStrict
+
+# webapp plugin tests
+-$ exists client/build/index.html
+-$ exists client/node_modules
+> webApp/npm:build
+$ exists client/build/index.html
+$ exists client/node_modules
+
+# webservice plugin tests
```

Comment:
ok, this is done. Want to take a look?

I want to try moving the cache key tests into their own test project for two reasons:

1. to see if we get a test run speedup (maybe scripted tests run in parallel - dunno)
2. to provide a second example for how to write a separate isolated test